### PR TITLE
[MNOE-573] - Send invoice to super admins

### PR DESCRIPTION
**Pull request by fgourichon**
@alachaum for review

### CODE REVIEW

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb [hunk 1]
```diff
@@ -134,6 +134,26 @@ def last_commission_amount
       render json: { last_commission_amount: format_money(tenant_billing) }
     end
 
+    # POST /mnoe/jpi/v1/admin/invoices/1/send_to_customers
+    def send_to_customers
+      invoice = MnoEnterprise::Invoice
+                  .select(:organization, organizations: [:id])
+                  .includes(:organization)
+                  .find(params[:id]).first
+
```
Comment (alachaum) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb:143]: you should add a `render_not_found`

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb [hunk 2]
```diff
@@ -134,6 +134,26 @@ def last_commission_amount
...
+      organization = MnoEnterprise::Organization
+                       .select(:orga_relations, orga_relations: [:id, :user_id, :role])
+                       .includes(:orga_relations)
+                       .find(invoice.organization.id).first
+
+      organization.orga_relations.each do |user|
+        if user.role == 'Super Admin'
```
Comment (alachaum) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb]: next unless `user.role == 'Super Admin'`

File: api/lib/mno_enterprise/concerns/mailers/system_notification_mailer.rb [hunk 3]
```diff
@@ -211,6 +211,29 @@ def access_approved(user_access_request_id)
     )
   end
 
+  def send_invoice(recipient_id, invoice_id)
+    recipient = MnoEnterprise::User.find_one(recipient_id)
```
Comment (alachaum) [inline api/lib/mno_enterprise/concerns/mailers/system_notification_mailer.rb]: You can reduce the number of fields on user.

File: core/app/models/mno_enterprise/invoice.rb [hunk 4]
```diff
@@ -8,6 +8,8 @@ class Invoice < BaseResource
     has_many :bills
     has_one :organization
 
+    custom_endpoint :send_to_customers, on: :member, request_method: :post
```
Comment (alachaum) [inline core/app/models/mno_enterprise/invoice.rb]: Remove

File: core/lib/mno_enterprise/smtp_client.rb [hunk 5]
```diff
@@ -7,6 +7,10 @@ class SmtpClient < ActionMailer::Base
     def deliver(template, from, to, vars={}, opts={})
       @info = vars
       @info[:company] = from[:name]
+  
+      for attachment in @info[:attachments]
```
Comment (alachaum) [inline core/lib/mno_enterprise/smtp_client.rb]: Replace by
```ruby
(@info[:attachments] || []).each do |attachment|
  attachments[attachment[:name]] = attachment[:value]
end
```

**Review by alachaum — APPROVED** [review]
👍

**Comment by fgourichon** [comment]
@alachaum changes implemented

**Pull request merged**